aea/test_tools/test_cases.py

```diff
@@ -734,6 +738,6 @@ def setup_class(cls):
     def teardown_class(cls):
         """Teardown the test class."""
         cls.path_to_aea = Path(".")
-        cls.packages_dir_path = Path("..", "packages")
+        # cls.packages_dir_path = Path("..", "packages")
```

Comment:
do we need a commented line here?

Reply:
For now, because this is only a quick fix. I did not want to remove this, can add todo